Cargo.toml

```diff
@@ -37,10 +37,9 @@ serde = { version = "1.0", default-features = false, optional = true }
 
 
 [dev-dependencies]
-rand = "0.6"
+secp256k1 = { path = ".", features = ["rand", "bitcoin_hashes", "recovery", "global-context"]}
```

Comment:
I don't think this is the "right" way to do this
Instead we should gate integration tests with the corresponding cfg (ie `#[cfg(feature = "recovery)]` over `ecdsa_sign_and_recover`)

Reply:
Oh yes, that's a good idea. Thanks